src/components/interaction-designer/BuilderCanvas.vue

```diff
@@ -23,10 +26,69 @@
 
     computed: {
       ...mapState('flow', ['flows']),
+
+      canvasHeight() {
+        const defaultHeight = window.screen.availHeight
+        if (!this.flows[0].blocks && this.flows[0].blocks.length) {
+          return defaultHeight
+        }
+
+        const blockAtTheLowestPosition = lodash.maxBy(this.flows[0].blocks, 'platform_metadata.io_viamo.uiData.yPosition')
+
+        if (!blockAtTheLowestPosition) {
+          return defaultHeight
+        }
+
+        const blockElement = document.getElementById(`block/${blockAtTheLowestPosition.uuid}`)
+
+        if (!blockElement) {
+          return defaultHeight
```

Comment:
Can we throw a `console.log()` onto this branch? It feels like this would be an odd state to get into! 
Something like:`log('Interaction Designer', 'Unable to find dom element corresponding to lowest block id.')`

Reply:
sure, added it